pkg/secrets/secrets.go

```diff
@@ -2,90 +2,68 @@ package secrets
 
 import (
 	"context"
-	"fmt"
 	"github.com/armory/go-yaml-tools/pkg/secrets"
-	"github.com/armory/spinnaker-operator/pkg/secrets/kubernetes"
-	"io/ioutil"
 	"os"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"strings"
 )
 
-// Decode decodes a potential value into a secret
-func Decode(ctx context.Context, val string) (string, error) {
-	return decode(decryptFunc, ctx, val)
+func init() {
+	secrets.Engines["kubernetes"] = NewSecretDecrypter
```

Comment:
Function name could be changed to something indicating it's a kubernetes specific decrypter, `NewKubeSecretDecrypter`?

Reply:
Oh yeah, it used to be in a different module. Will rename